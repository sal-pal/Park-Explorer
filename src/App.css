.App {
  text-align: center;
  position: relative;
  top: 100%;
}

.formElem {
    display: block;
    max-width: 300px;
    margin: 0px auto
}

a {
    font-size: 12px;   
}