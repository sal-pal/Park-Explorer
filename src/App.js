import React, { Component } from 'react'
import ReactDOM from 'react-dom'
import './App.css'

class App extends Component {  
  render() {
    return (
      <div className="App">
        
        <h1> 
           Welcome to Park Forum! <br/>
           Please Log In
        </h1>
        
        <input type="text" className="formElem" placeholder="Username"/>
        <input type="text" className="formElem" placeholder="Password"/>
        <button className="formElem">Sumbit</button>
        <a href="#">Sign Up</a>
      </div>
    );
  }
}



function run() {
    ReactDOM.render(<App/>, document.getElementById('root'));
}

if (window.addEventListener) {
    window.addEventListener('DOMContentLoaded', run);
} 
else {
    window.attachEvent('onload', run);
}
