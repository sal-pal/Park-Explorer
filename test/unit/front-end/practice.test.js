import React from 'react';
import App from '../../src/front-end/App.js';
import renderer from 'react-test-renderer';

test('snapshot can be taken', () => {
    const component = renderer.create(
        <App/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})