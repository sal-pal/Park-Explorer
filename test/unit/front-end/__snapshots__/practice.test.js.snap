exports[`test snapshot can be taken 1`] = `
<div
  className="App">
  <p
    className="App-intro">
    To get started, edit 
    <code>
      src/App.js
    </code>
     and save to reload.
  </p>
</div>
`;
